source/include/core_http_client.h

```diff
@@ -672,6 +744,46 @@ HTTPStatus_t HTTPClient_AddRangeHeader( HTTPRequestHeaders_t * pRequestHeaders,
  * - #HTTP_SECURITY_ALERT_INVALID_STATUS_CODE
  * - #HTTP_SECURITY_ALERT_INVALID_CHARACTER
  * - #HTTP_SECURITY_ALERT_INVALID_CONTENT_LENGTH
+ *
+ * <b>Example</b>
```

Comment:
Would the markdown equivalent work here too? \*\*Example\*\*

Reply:
sure